ansible_collections/arista/avd/roles/eos_designs/python_modules/base/avdstructuredconfig.py

```diff
@@ -715,6 +715,8 @@ def ptp(self) -> dict | None:
                 },
             },
         }
+        if ptp["mode"] == "boundary" and (mode_one_step := get(self.shared_utils.switch_data_combined, "ptp.mode_one_step", default="false")) is True:
+            ptp.update({"mode_one_step": mode_one_step})
```

Comment:
You could just add the "mode_one_step" in the large dict above without a default value, since we only support "boundary".

Reply:
Fixed